### DISCUSSION TITLE
Stats API Timeseries - Add hourly breakdown for a longer period

### DISCUSSION THREAD

**Discussion by RobertJoonas** [plausible/analytics#1434] [Anything Else]
Currently, from the Stats API Timeseries endpoint, it's only possible to get an hourly breakdown with a period of one day.

Would be nice if an hourly interval would also be supported for the periods 7d, month (perhaps even 6mo or 12mo?)

**Comment by TimDaub** [comment]
currently this is what I have to do: https://github.com/rugpullindex/plausible-crawler/commit/37b0aba9b930cd987748b2fcd169d74c0da45047

- iterate over all days that I want for an hourly overview
- As there's only 600 requests per hour on the stats API, I ended up getting rate limited too.
- It'd be cool if I could just download the hourly data via a designated endpoint as I felt I did something "inappropriate" to your api by spamming it with so many requests
- but your servers were nice to me and I didn't end up having any problems. The crawler is a good work around for now
